flow/connectors/mongo/mongo.go

```diff
@@ -167,6 +168,11 @@ func parseAsClientOptions(config *protos.MongoConfig, meteredDialer utils.Metere
 		clientOptions.SetTLSConfig(tlsConfig)
 	}
 
+	// For debugging only: set env var MONGO_COMMAND_MONITOR to "true" to log all change stream commands
+	if os.Getenv("MONGO_COMMAND_MONITOR") == "true" {
```

Comment:
There's also [PEERDB_LOG_LEVEL](https://github.com/PeerDB-io/peerdb/pull/3647) that we're consuming when printing PG messages in debug only